src/main/java/com/Crowdsourcing/messages/CrowdsourcingMessages.java

```diff
@@ -143,6 +143,10 @@ public class CrowdsourcingMessages
 	private static final int CHARM_OF_PREPARATION = 12100;
 	private static final int CHARM_OF_BOOST = 12104;
 
+  // Hallowed Sepulchre coffins
+  private static final String SEPULCHRE_FAILURE = "You have been poisoned.";
```

Comment:
This is generic and is going to cause us to add metadata to a bunch of stuff unrelated to these chests. It looks like the volume of messages is low enough (~9k per day) that it's probably not that big of a deal from a data perspective, but we will need to account for this when querying the data. We get location with all these but I am not sure how reliable that will be for instances.

Does this fire even when you are already poisoned?

Also it seems to use an exclamation mark based on the data and [this vid (timestamp)](https://youtu.be/wTaPmv0aaII?si=llbVJm_whDQp1kPL&t=450).
```suggestion
  private static final String SEPULCHRE_FAILURE = "You have been poisoned!";
```

Reply:
> Does this fire even when you are already poisoned?

I don't really have a way to test this easily. The failure rate for me (92 thieving, 86 agility) seems to be around 10%. So getting two back to back poisonings without a success would take a very long time to see. I feel like it is rare enough to not skew the data much.

Reply:
Any rough idea of success at lower levels? I think you might be underestimating how much this will skew the data if it scales poorly at lower levels. You likely wont get much data at lower levels here so that makes any missing data more concerning. If we are going to get rates we should feel confident theyre correct.

You dont need to fail twice in a row btw, you could just not clear poison between chests (or go in poisoned to test). Icr if the statues clear it at the end of floors.

Reply:
I confirmed a double failure gives the message "You trigger a trap on the chest which poisons you!" So it is the same if you fail while poisoned or fail while under anti-poison protection.